rust/gui-client/src-tauri/Cargo.toml

```diff
@@ -59,12 +59,12 @@ futures = "0.3.30"
 
 [target.'cfg(target_os = "windows")'.dependencies]
 tauri-winrt-notification = "0.1.3"
-windows-implement = "0.52.0"
+windows-implement = "0.53.0"
```

Comment:
They don't seem to have windows-implement 0.54.0 yet. I'm not sure how their versioning works.

Reply:
Hm, if this might cause an issue we could group these dependencies in `dependabot.yml` so Dependabot bumps them in unison?

Reply:
So that's how those work. I updated it just now, is there a trick to make sure it's valid?

Reply:
https://github.com/firezone/firezone/pull/3919/checks?check_run_id=22266328723